### PR TITLE
Standardized README

### PR DESCRIPTION
**Pull request by RichardLitt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
 
 > logging helpers for go-libp2p
 
-## Documenation
+## Install
 
-See https://godoc.org/github.com/ipfs/go-libp2p-loggables.
+```
+go get github.com/libp2p/go-libp2p-loggables
+```
```
Comment (Kubuxu) [inline README.md:14]: It is library, you can't really install it but if we do this in other lib only repos too, then let's keep it.

**Comment by Kubuxu** [comment]
One comment and LGTM

File: README.md (hunk 1 above)
Comment (RichardLitt) [reply README.md:14]: What does one do with a go library? Do you import it? Or is it referenced?

Comment (RichardLitt) [reply README.md:14]: Should this be compiled from source? For instance:

```
git clone git://github.com/libp2p/go-libp2p-loggables.git
cd go-libp2p-loggables && go build
```

I looked around at a few libraries on [this list](http://go-lang.cat-v.org/pure-go-libs), and they all seem to think that the installation should be `go get`. What am I missing?

Comment (Kubuxu) [reply README.md:14]: Ok, lets leave it as it is.

**Comment by Kubuxu** [comment]
LGTM

**Pull request merged**